internal/services/data_record/resource_data_record.go

```diff
@@ -58,31 +60,36 @@ func (r *DataRecordResource) Schema(ctx context.Context, req resource.SchemaRequ
 				Delete: true,
 				Read:   true,
 			}),
+			"disable_on_destroy": schema.BoolAttribute{
+				MarkdownDescription: "If true, the resource will state",
```

Comment:
Need a better description.  This one doesn't make sense to me.

If true, the resource will either set isdisabled or state with a PATCH before attempting to DELETE the record on destroy.

Or something like that.